vignettes/dm-introduction-relational-data-models.Rmd

```diff
@@ -75,9 +76,6 @@ This can result in long and inflated pipe chains full of `left_join()`, `anti_jo
 In classical {dplyr} notation you need three `left_join()` calls to merge the `flights` table gradually to `airlines`, `planes` and `airports` tables to create one wide data frame.
 
 ```{r}
```

Comment:
It's better to keep these visible and also load nycflights13 here (but also add to the setup chunk so that the startup messages are hidden).